baqend/_updateMultiComparison.ts

```diff
@@ -162,7 +155,7 @@ export async function updateMultiComparison(db: baqend, bulkTestRef: model.BulkT
 
     bulkTest.speedKitMeanValues = new db.Mean(aggregateFields(pickResults(bulkTest, 'speedKit'), fields))
     bulkTest.competitorMeanValues = new db.Mean(aggregateFields(pickResults(bulkTest, 'competitor'), fields))
-    bulkTest.factors = aggregateBulkTestFactors(db, bulkTest)
+    bulkTest.factors = calcMeanFactors(db, bulkTest.speedKitMeanValues, bulkTest.competitorMeanValues)
```

Comment:
I think this is not what we are looking for exactly. We are interested in the factors of the second best `TestOverview`. If I understand correctly, in this example, we are calculating the mean values for Speed Kit and competitor separately, in which case the values cannot be compared anymore. What we need to do instead is sort all Comparisons by their FMP factor and then pic the second best. From this single comparison, we can calculate all factors for the bulktest